identityClient/identityClient_test.go

```diff
@@ -13,15 +13,15 @@ import (
 )
 
 var (
-	toznyCyclopsHost            = os.Getenv("TOZNY_CYCLOPS_SERVICE_HOST")
-	internalIdentityServiceHost = os.Getenv("E3DB_IDENTITY_SERVICE_HOST")
-	e3dbAuthHost                = os.Getenv("E3DB_AUTH_SERVICE_HOST")
-	e3dbAccountHost             = os.Getenv("E3DB_ACCOUNT_SERVICE_HOST")
-	e3dbAPIKey                  = os.Getenv("E3DB_API_KEY_ID")
-	e3dbAPISecret               = os.Getenv("E3DB_API_KEY_SECRET")
-	e3dbClientID                = os.Getenv("E3DB_CLIENT_ID")
-	bootstrapPublicSigningKey   = os.Getenv("BOOTSTRAP_CLIENT_PUBLIC_SIGNING_KEY")
-	bootstrapPrivateSigningKey  = os.Getenv("BOOTSTRAP_CLIENT_PRIVATE_SIGNING_KEY")
+	toznyCyclopsHost            = utils.MustGetenv("TOZNY_CYCLOPS_SERVICE_HOST")
```

Comment:
imo, unlike production services, tests don't need to fail for missing environment variables. For something like this which is only ran by developers, I'd prefer leaving it open to being able to only specify in your local environment the variables needed for one's own testing.

Reply:
I actually hit an error where tests would simply not run because of missing environment variables. I'm OK having them optional if tests don't break, but it took a bit of work to figure out what was actually missing. I'm fine dropping back to `os.` level.

Reply:
welp, I suppose better the devil you know. Local developers can always comment out what ones they don't need, or when adding new ones in the future optionally make it a MustGet